### ISSUE TITLE
Make submit button of mass action translatable.

### ISSUE THREAD

**Issue by michael-arif** [bagisto/bagisto#2307]
# Bug report

### Title
Make submit button of mass action translatable.

### Issue Description
All pages that can perform mass action has a button titled Submit. It's not yet translatable.

### Preconditions
    1. framework Version: 0.2.2

**Comment by michael-arif** [comment]
Hi @jitendra-webkul 

Can you please let me know where should I call packages/Webkul/Ui/src/Resources/views/datagrid/partials/mass-action-header.blade.php from?

**Comment by jitendra-webkul** [comment]
@michael-arif 

Can you elaborate more?

**Comment by michael-arif** [comment]
@jitendra-webkul 

Thank you for replying.

In the Issue #2307 fixed #2348, there are 5 files changed and 1 of them is:
packages/Webkul/Ui/src/Resources/views/datagrid/partials/mass-action-header.blade.php
The solution to my question is in line 28-30. However, I just could not find mass-action-header.blade.php in my bagisto installation. So, I asked the above question.

After spending a day searching for the content similar to the mass-action-header.blade.php,
I found that it is part of packages/Webkul/Ui/src/Resources/views/datagrid/table.blade.php.
I adjusted line 176 as follow:
<input type="submit" value="{{ __('ui::app.datagrid.submit-button-title') }}" class="btn btn-sm btn-primary" style="margin-left: 10px;">
Now, the blue Submit button that appears to the right of the mass action combobox (bulk delete, bulk update status) is translatable. Thank you very much.
